src/style.css

```diff
@@ -1,3 +1,100 @@
-body {
+/* stylelint-disable-next-line import-notation */
```

Comment:
Avoid disabling lint rules if you can
In this very case, this setting is not needed (probably some leftover)